### ISSUE TITLE
Update options in generic_runner.py

### ISSUE THREAD

**Issue by XinhuiLi** [trendscenter/MISA-pytorch#5]
Change batch size in generic_runner.py to =< the number of samples.

For neuroimaging data, we could set the default options to be: batch size = 128, learning rate = 0.001, epochs = 500.

**Comment by rsilva8** [comment]
Update this [config file](https://github.com/trendscenter/MISA-pytorch/blob/main/configs/sim-siva.yaml)

Right before this [line](https://github.com/trendscenter/MISA-pytorch/blob/main/runners/generic_runner.py#L92), test number of rows in ds.mat_data is >= batch_size. Else, set batch_size = number of rows in ds.mat_data

**Comment by XinhuiLi** [comment]
The number of samples can be retrieved from the Dataset object `ds` using `ds.__len__()`
